perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -93,7 +93,7 @@
     'MaxLatency(ms)': max}
 
 
-flags.DEFINE_string('ycsb_version', '0.9.0', 'YCSB version to use. Defaults to '
+flags.DEFINE_string('ycsb_version', '0.13.0', 'YCSB version to use. Defaults to '
                     'version 0.9.0.')
```

Comment:
Update comment to refer to 0.13.0. +jellyfishcake as FYI.

Reply:
this was for a workaround since 0.9.0 was not downloading, i changed back to default.